kumascript2/macros/A11yRoleQuicklinks.ejs

```diff
@@ -0,0 +1,26 @@
+<%
+// Inserts quicklinks for the accessibility roles.
+var lang = env.locale;
+var url = "";
+
+switch(lang) {
+    case "de":
+        url = "/de/docs/Web/Accessibility/AT-APIs/Gecko/Rollen";
+        break;
+    case "fr":
+        url = "/fr/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
+        break;
+    case "ja":
+        url = "/ja/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
+        break;
+    case "ru":
+        url = "/ru/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
+        break;
+    default:
+        url = "/en-US/docs/Web/Accessibility/AT-APIs/Gecko/Roles";
+        break;
+}
+%>
+<section class="Quick_links" id="Quick_Links">
+<%-await wiki.tree(url, 2, 0, 0, 1)%>
```

Comment:
You should be able to ignore the macro files. Since this PR is against Kuma, these are copies of them not just the changed lines.

The primary change is that we now use the `await` keyword before all async (or possbily async) function calls.

Reply:
At this point in the review skip way down to kumascript2/src/